### ISSUE TITLE
Not Windows friendly :(

### ISSUE THREAD

**Issue by trullock** [tonsky/FiraCode#17]
Doesn't seem to be antialiased properly and the ligatures dont seem to work :(

**Comment by tonsky** [comment]
I have no idea how to fix it, sorry

Editor support is not universal, check out this list: https://github.com/i-tu/Hasklig#editor-support

I have at least VS2013 confirmed as working:
<img src=http://habrastorage.org/files/2b5/151/a43/2b5151a438994de599413cea91817894.png>

**Comment by trullock** [comment]
Really? Oh, thats exactly what I was trying it in. Let me try again... is that the ttf or the otf working in VS?

**Comment by tonsky** [comment]
No idea, that screenshot isn’t mine. In general OTF is preferred, but you
better try both.

**Comment by trullock** [comment]
Right, so it turns out VS needs a restart for the font to work, despite detecting it in its list for you to select from. I'll close this then
